packaging/dockerfiles/Dockerfile.rocky9

```diff
@@ -9,19 +9,17 @@ RUN true \
   && dnf config-manager --set-enabled crb \
   && dnf upgrade -y \
   && dnf install -y \
-    gcc-c++ \
-    java-17-openjdk-devel \
-    mpich-devel \
-    nano \
-    ninja-build \
-    rpm-build \
-    wget \
+  gcc-c++ \
+  java-17-openjdk-devel \
+  mpich-devel \
+  nano \
+  ninja-build \
+  rpm-build \
   && dnf clean all \
   && rm -rf /var/cache/dnf/ \
   # Note: We install cmake from kitware's site to have a more recent version \
   # If rocky9 eventually supports cmake >= 3.22 we may use dnf again \
-  && wget "https://github.com/Kitware/CMake/releases/download/v3.26.5/cmake-3.26.5-linux-x86_64.sh" \
+  && curl -LO "https://github.com/Kitware/CMake/releases/download/v3.26.5/cmake-3.26.5-linux-x86_64.sh" \
   && sh cmake-3.26.5-linux-x86_64.sh --prefix=/usr/local/ --exclude-subdir \
   && rm -f cmake-3.26.5-linux-x86_64.sh \
-  && dnf remove -y wget "dnf-command(config-manager)"\
```

Comment:
I think we should keep the removal of `dnf-command(config-manager)` which is needed only to install `ninja-build`.